astropy/logger.py

```diff
@@ -19,6 +19,14 @@
 
 __all__ = ['Conf', 'conf', 'log', 'AstropyLogger', 'LoggingError']
 
+# import the logging levels from logging so that one can do:
+# log.setLevel(log.DEBUG), for example
+logging_levels = ['DEBUG', 'CRITICAL', 'FATAL', 'ERROR', 'WARN', 'WARNING',
```

Comment:
(As far as listing them in order goes I think it's helpful just for documentation purposes).